prime-router/metadata/fhir_mapping/hl7/secondary/PatientContact.yml

```diff
@@ -65,6 +65,13 @@ telecom_business:
   constants:
     use: work
 
+period:
```

Comment:
Just to confirm, this is here to test converting to Datetime?

Reply:
Specifically, this is to test converting to DateTime from a HL7 DT (Date), instead of a DTM (DateTime). DT is pretty uncommon in ORU_R01, so even though it's kind of a compound usage within `period`, this was the simplest case I could find.